### PR TITLE
SCB-1558 support query history by key and fix wrong API doc

### PR DESCRIPTION
**Pull request by tianxiaoliang**

### CODE REVIEW

File: pkg/model/kv.go [hunk 1]
```diff
@@ -17,6 +17,15 @@
 
 package model
 
+//KVRequest is http request body
+type KVRequest struct {
+	Key       string            `json:"key" yaml:"key"`
+	Value     string            `json:"value,omitempty" yaml:"value,omitempty"`
+	ValueType string            `json:"value_type,omitempty" bson:"value_type,omitempty" yaml:"value_type,omitempty"` //ini,json,text,yaml,properties
+	Checker   string            `json:"check,omitempty" yaml:"check,omitempty"`                                       //python script
```
Comment (surechen) [inline pkg/model/kv.go]: Why use python script?

File: server/resource/v1/history_resource.go [hunk 2]
```diff
@@ -33,16 +33,17 @@ import (
...
+//GetRevisions search key only by label
+func (r *HistoryResource) GetRevisions(context *restful.Context) {
 	var err error
 	labelID := context.ReadPathParameter("label_id")
 	if labelID == "" {
 		openlogging.Debug("label id is null")
 		WriteErrResponse(context, http.StatusForbidden, "label_id must not be empty", common.ContentTypeText)
 		return
 	}
-	revisions, err := service.HistoryService.GetHistoryByLabelID(context.Ctx, labelID)
```
Comment (surechen) [inline server/resource/v1/history_resource.go]: Are there some condition parameters will be added in querying history resources later？For example time spans.

File: server/resource/v1/history_resource.go [hunk 3]
```diff
@@ -33,16 +33,17 @@ import (
...
+	key := context.ReadQueryParameter("key")
```
Comment (surechen) [inline server/resource/v1/history_resource.go]: Are there any other condition parameters will be added in querying history resources later？For example time spans.

File: server/service/mongo/history/service.go [hunk 4]
```diff
@@ -20,15 +20,30 @@ package history
...
+	if opts.Key != "" {
+		filter = bson.M{
+			"label_id": labelID,
+			"data.key": opts.Key,
+		}
+
+	} else {
+		filter = bson.M{"label_id": labelID}
+	}
 	return getHistoryByLabelID(ctx, filter)
```
Comment (surechen) [inline server/service/mongo/history/service.go]: Whether there is defensive code, such as setting the maximum number of queries returned？

Comment (tianxiaoliang) [reply server/service/mongo/history/service.go]: yes, should develope pagination in future

File: server/resource/v1/history_resource.go (hunk 3 above)
Comment (tianxiaoliang) [reply server/resource/v1/history_resource.go]: will develop in future

File: pkg/model/kv.go (hunk 1 above)
Comment (tianxiaoliang) [reply pkg/model/kv.go]: it is a TODO, this python scrript will be a function which validate the value

**Comment by tianxiaoliang** [comment]
plz open 2 issues for pagination and time span, and assign to me

**Comment by ChinX** [comment]
LGTM

**Pull request merged**